lib/elixirlight/colorchanger.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Elixirlight.Colorchanger do
+  import ColorUtils
+
+  @doc """
+  Initializes a white color upon startup
+  Using RGB here cuz that is what our device uses
+  """
+  def start_link do
+    color = %ColorUtils.RGB{red: 255, green: 255, blue: 255}
```

Comment:
By importing the utils, you don't need to use the `ColorUtils` here you can shorten it to: `%RGB{`